class.application-passwords.php

```diff
@@ -228,6 +244,56 @@ public static function create_new_application_password( $user_id, $name ) {
 	}
 
 	/**
+	 * Create application password via ajax
+	 *
+	 * @since 0.1-dev
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function ajax_create_application_password() {
+		$user_id = get_current_user_id();
+		$name = $_POST['app_name'];
+		if ( ! isset( $_POST['create_application_password'] ) || wp_verify_nonce( $_POST['_nonce_create_application_password'], 'create_application_password') ) {
+			$res = self::create_new_application_password( $user_id, $name );
+
+			if ( $res ) {
+				wp_send_json_success( array( 'new_pass' => $res ) );
+				wp_die();
+			}
+		}
+
+		wp_send_json_error();
+		wp_die();
+
+	}
+
+	/**
+	 * Delete application passwords via ajax
+	 *
+	 * @since 0.1-dev
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function ajax_delete_application_password() {
+		$user_id = get_current_user_id();
+		if ( ! empty( $_POST['delete_application_password'] ) ) {
+			$slug = $_POST['delete_application_password'];
+			check_admin_referer( "delete_application_password-{$slug}", '_nonce_delete_application_password' );
+
+			$res = self::delete_application_password( $user_id, $slug );
+		}
+
+		if ( true === $res ) {
+			wp_send_json_success();
+		}
+
+		wp_send_json_error();
+		wp_die();
```

Comment:
`wp_die` is redundant because `wp_send_json` the function that `wp_send_json_success` & `wp_send_json_error` uses does this automatically when `DOING_AJAX`.